### PR TITLE
[UXE-3667] feat: implemented the focus error feature in the view network lists

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
### What is the new behavior introduced by this PR?
* focus error in network lists fields
* migrate inputs to our field blocks

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
no

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [x] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/views/NetworkLists/FormFields/FormFieldsCreateNetworkLists.vue [hunk 1]
```diff
@@ -73,118 +72,66 @@
...
-        <Dropdown
-          appendTo="self"
-          :class="{ 'p-invalid': networkListTypeError }"
-          v-model="networkListType"
+        <FieldDropdown
+          label="Type"
+          name="networkListType"
           :options="options"
-          optionLabel="name"
+          @change="resetRegionAndCity"
```
Comment (HerbertJulio) [inline src/views/NetworkLists/FormFields/FormFieldsCreateNetworkLists.vue:97]: Pode remover esse @change

File: src/views/NetworkLists/FormFields/FormFieldsCreateNetworkLists.vue [hunk 2]
```diff
@@ -73,118 +72,66 @@
...
-        >
-        <TextareaComponent
-          :class="{ 'p-invalid': asnError }"
-          v-model="asn"
+        <FieldTextArea
+          label="List *"
+          placeholder="1234&#10;4321"
+          name="asn"
           rows="2"
           cols="30"
```
Comment (HerbertJulio) [inline src/views/NetworkLists/FormFields/FormFieldsCreateNetworkLists.vue:114]: o cols por default é 30 então pode remove

File: src/views/NetworkLists/FormFields/FormFieldsCreateNetworkLists.vue [hunk 3]
```diff
@@ -73,118 +72,66 @@
...
-        <TextareaComponent
-          :class="{ 'p-invalid': ipCidrError }"
-          v-model="ipCidr"
+        <FieldTextArea
+          label="List *"
+          placeholder="185.241.208.232&#10;194.26.192.64&#10;171.25.193.25 #comment"
+          name="ipCidr"
           rows="16"
-          id="ipCidr"
           cols="30"
```
Comment (HerbertJulio) [inline src/views/NetworkLists/FormFields/FormFieldsCreateNetworkLists.vue:129]: o mesmo neste ponto

File: src/views/NetworkLists/FormFields/FormFieldsCreateNetworkLists.vue (hunk 1 above)
Comment (peterpaulo-azion) [inline src/views/NetworkLists/FormFields/FormFieldsCreateNetworkLists.vue:97]:
```suggestion
```

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**